frontend/src/app/Settings/Facility/FacilityForm.tsx

```diff
@@ -47,53 +47,49 @@ const FacilityForm: React.FC<Props> = (props) => {
   };
 
   return (
-    <main className="prime-home">
-      <div className="grid-container">
```

Comment:
this diff looks big, but the only difference is I removed these two lines. These are common lines in all the settings pages, so I moved them to`Settings.tsx`